packages/fxa-auth-server/config/index.js

```diff
@@ -552,6 +552,21 @@ const conf = convict({
       default: '',
     },
   },
+  profileServer: {
+    url: {
+      doc: 'The url of the corresponding fxa-profile-server instance',
+      env: 'PROFILE_SERVER_URL',
+      format: 'url',
+      default: 'http://127.0.0.1:1111',
```

Comment:
I think we usually prefer to set the defaults for prod and then override them for local, dev and stage.

Reply:
Updated to change this to `https://profile.accounts.firefox.com` - I think that's the prod URL here